provider/datagen/src/registry.rs

```diff
@@ -139,9 +139,7 @@ registry!(
     icu_calendar::provider::JapaneseErasV1Marker = "calendar/japanese@1",
     icu_calendar::provider::JapaneseExtendedErasV1Marker = "calendar/japanext@1",
     icu_calendar::provider::WeekDataV1Marker = "datetime/week_data@1",
-    #[cfg(feature = "icu_casemap")]
```

Comment:
change this to `#[cfg(any(all(), feature = "icu_casemap"))]` for consistency. You also don't need it for both markers.

Reply:
wait, why not on both markers?

Reply:
oh because of the macro